src/main/webapp/app/shared/user-settings/user-settings-guide.txt

```diff
@@ -0,0 +1,32 @@
+Developer Guide/Readme for using and extending user settings
+
+    The (data/interface/class( hierarchy is explained in user-settings.service.ts (look at the interfaces)
+
+    How to create a new "child"-settings.component : (Look at notification-settings as a prime example for child-settings):
+
+        0)  Add a new route for the new settings under userSettingsState (children)
```

Comment:
step 0?

Reply:
I changed it so the index starts with 1 not 0 anymore